src/core/qgshttptransaction.cpp

```diff
@@ -486,6 +497,19 @@ void QgsHttpTransaction::dataStateChanged( int state )
 }
 
 
+void QgsHttpTransaction::ignoreSslErrors( const QList<QSslError> &errors )
+{
+for ( const QSslError e : errors )
+  {
+    QgsDebugMsg( e.errorString() );
```

Comment:
Can this be replaced with something that's visible in release mode too (QgsMessageLog? What's used on server?)